packages/front-end/components/Archetype/SimulateFeatureValues.tsx

```diff
@@ -0,0 +1,516 @@
+                                key={feature.id + en.id + "row"}
+                                className="position-relative  cursor-pointer"
+                              >
+                                <div>
+                                  {ArchetypeValueDisplay({
+                                    result: res,
+                                    feature,
+                                  })}
+                                </div>
+                              </td>
+                            );
+                          })}
+                        </>
+                      )}
+                    </tr>
+                  </Fragment>
+                );
+              })}
+              {!items.length && (
+                <tr>
+                  <td colSpan={numColumns}>No matching features</td>
+                </tr>
+              )}
+            </tbody>
+          </table>
+          {Math.ceil(items.length / NUM_PER_PAGE) > 1 && (
+            <Pagination
+              numItemsTotal={items.length}
+              currentPage={currentPage}
+              perPage={NUM_PER_PAGE}
+              onPageChange={(d) => {
+                setCurrentPage(d);
+              }}
+            />
+          )}
+        </div>
+      </div>
+      <div className="alert-info mt-5 p-3 cursor-pointer align-items-center">
+        <div
+          className="d-flex"
+          onClick={(e) => {
+            e.preventDefault();
+            setOpenWarning(!openWarning);
+          }}
+        >
+          <div className="p-2 pr-3">
+            <FiAlertTriangle />
+          </div>
+          <div>
+            These results use the JS SDK, which supports the V2 hashing
+            algorithm. If you use one of the older or unsupported SDKs, you may
+            want to change the hashing algorithm of the experiment to v1 to
+            ensure accurate results.
+          </div>
+          <div className="p-2">
+            <FaChevronRight
+              style={{
+                transform: `rotate(${openWarning ? "90deg" : "0deg"})`,
+              }}
+            />
+          </div>
+        </div>
+        {openWarning && (
+          <div className="p-3">
+            The following SDK versions support V2 hashing:
+            <MinSDKVersionsList capability="bucketingV2" />
+          </div>
+        )}
+      </div>
+    </>
+  );
+
+  return (
+    <>
+      {editAttributesModalOpen && (
+        <SimulateFeatureModal
+          archetype={archetype}
+          archetypeMap={archetypeMap}
+          attributes={attributes}
+          close={() => {
+            setEditAttributesModalOpen(false);
+          }}
+          onSubmit={({ archetype, attributes }) => {
+            setArchetype(archetype);
+            setAttributes(attributes);
+            setEditAttributesModalOpen(false);
+            refreshResults();
+          }}
+        />
+      )}
+      <div className="container-fluid pagecontents pt-4">
```

Comment:
consistent margins between 2 tabs
```suggestion
      <div className="pagecontents">
```

Reply:
done